oem/ibm/libpldmresponder/file_io_by_type.cpp

```diff
@@ -39,7 +39,8 @@ void FileHandler::dmaResponseToRemoteTerminus(
     const SharedAIORespData& sharedAIORespDataobj,
     const pldm_completion_codes rStatus, uint32_t length)
 {
-    Response response(sizeof(pldm_msg_hdr) + sharedAIORespDataobj.command, 0);
+    Response response(
```

Comment:
once this PR goes in I guess we can remove this change - https://github.com/ibm-openbmc/pldm/pull/598

Reply:
done